src/Umbraco.Infrastructure/Migrations/NoopMigration.cs

```diff
@@ -1,14 +1,12 @@
+
 namespace Umbraco.Cms.Infrastructure.Migrations;
 
-public class NoopMigration : MigrationBase
+public class NoopMigration : AsyncMigrationBase
```

Comment:
This could be reverted back to `MigrationBase` to avoid an extra breaking change (so the base class doesn't change)... We have similar base class changes in `UnscopedMigrationBase` and `PackageMigrationBase`, which also don't inherit from `MigrationBase` anymore, but can't be fixed (because you can't simultaneously inherit from two base classes).